### ISSUE TITLE
Request for opcode assignment for `memory.copy` and `memory.fill`

### ISSUE THREAD

**Issue by julian-seward1** [WebAssembly/bulk-memory-operations#9]
What should the binary encoding of `memory.copy` and `memory.fill` look like?  Specifically:

* should there be opcodes allocated in the top level opcode space?  If so, what values?

* if not, will there be an escape into a new space, with its own opcodes there?  If so what values for the opcodes and the escape?

**Comment by jfbastien** [comment]
We should follow the resolution of the polls from https://github.com/WebAssembly/meetings/blob/master/2017/CG-05.md (search for `0xf` to see the full discussion and reversal).

**Comment by binji** [comment]
Yeah, these instructions don't really fit with any of the prefixes we have talked about currently: math=`0xfc`, simd=`0xfd`, threads=`0xfe`. We can allocate another prefix, or we could try to fill in some of the holes.

My guess is that these opcodes won't be very common, so we shouldn't use top-level. OTOH, there probably won't be many other instructions like this (I think?). If not, then perhaps using a few top-level will be better than allocating a new prefix.

The total instruction count (currently) for this proposal is 7:

* `memory.init`
* `memory.drop`
* `memory.copy`
* `memory.fill`
* `table.init`
* `table.drop`
* `table.copy`

We've talked about adding `table.fill` as well, to make 8. The top-level space has the following inclusive ranges unallocated (by MVP or another proposal):

* `0x12`..`0x19`: 8 available
* `0x27`: 1 available
* `0xc5`..`0xef`: ~43 available (not clear what anyref proposal will use)

**Comment by julian-seward1** [comment]
A two-level allocation sounds reasonable to me.  What is the process for getting some actual numbers, at least as a preliminary assignment?

**Comment by binji** [comment]
> Maybe it's something we can discuss at the next CG video call?

See https://github.com/WebAssembly/meetings/pull/194

**Comment by julian-seward1** [comment]
@binji, thanks for the 0xfc prefix clarification.  Was there any assignment made for the second level bytes, at least for the .copy and .fill operations?
